org.grisbi.Grisbi.yml

```diff
@@ -0,0 +1,62 @@
+app-id: org.grisbi.Grisbi
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.gnome.Sdk.Locale
+command: grisbi
+
+rename-desktop-file: grisbi.desktop
+
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --filesystem=host
+
+cleanup:
+  - '/include'
+  - '*.a'
+
+modules:
+
+  - shared-modules/intltool/intltool-0.51.json
+
+  - name: libgsf
+    sources:
+      - type: archive
+        url: https://gitlab.gnome.org/GNOME/libgsf/-/archive/LIBGSF_1_14_50/libgsf-LIBGSF_1_14_50.tar.bz2
+        sha256: da0231e013a07b475d1c1f33d6f4c4ba62a8ae700b648d00c70265d2e482a984
+
+  - name: opensp
+    config-opts:
+      - --disable-doc-build
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/project/openjade/opensp/1.5.2/OpenSP-1.5.2.tar.gz
+        sha256: 57f4898498a368918b0d49c826aa434bb5b703d2c3b169beb348016ab25617ce
+
+  - name: libofx
+    config-opts:
+      - --with-opensp-includes=/app/include/OpenSP
+      - --with-opensp-lib=/app/lib
+      - --disable-gengetopt
+      - --disable-tools
+    sources:
+      - type: archive
+        url: https://github.com/libofx/libofx/archive/refs/tags/0.10.9.tar.gz
+        sha256: ea9fa07759622ecc7f25b637fa8fb34d587af80607ca4389d25966a6a4f94ab9
+
+  - name: grisbi
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/project/grisbi/grisbi%20stable/2.0.x/2.0.5/grisbi-2.0.5.tar.bz2
+        sha256: bd3adbabfc4b4dfc05eff62d2b36458a24b0f00d07cf35a29f6af2f203c77a3f
+      - type: file
+        path: grisbi.metainfo.xml
+    post-install:
+      - install -m 644 -p -D grisbi.metainfo.xml /app/share/metainfo/org.grisbi.Grisbi.metainfo.xml
+      - mv /app/share/icons/hicolor/scalable/apps/grisbi.svg /app/share/icons/hicolor/scalable/apps/org.grisbi.Grisbi.svg
```

Comment:
use `rename-icon`.

Reply:
I tried, but it kept saying `Error: icon grisbi.svg not found below .flatpak-builder/rofiles/rofiles-0JifHG/files/share/icons`. I used `rename-icon: grisbi.svg`.

Reply:
because it's the icon **name**. Icon are supported in multiple formats.

>        rename-icon (string)
>           Any icon with this name will be renamed to a name based on id during the cleanup phase. Note that this is the icon name, not the full
>           filenames, so it should not include a filename extension.